php-classes/Slate/CBL/Tasks/TasksRequestHandler.php

```diff
@@ -377,7 +377,7 @@ public static function updateGoogleFilePermissions(Task $Record)
                                 'File' => $DriveFile,
                                 'Context' => $StudentTask,
                                 'ParentAttachment' => $attachment,
-                                'RevisionID' => 1
+                                'FileRevisionID' => 1
```

Comment:
@nbey do we not have access to the actual revision ID? Does `1` pass string validation?